### DISCUSSION TITLE
Drop down list on long tap

### DISCUSSION THREAD

**Discussion by t214c** [iSoron/uhabits#1245] [Feature Requests] [answered]
This [app](https://play.google.com/store/apps/details?id=com.pavlenko.Habo) has a nice way of entering a habit status for the day, which is a drop down list of the available options.

This way is easier and faster instead of cycling the options one by one (especially with 2.1 which adds the needed "?" check to the cycled states. It works fine with the long press option to prevent accidental edits.

**Comment by hiqua** [comment, marked as answer]
I think that's already the case with the latest version on `dev`, but let's double-check when the next version is released.

**Reply by t214c** [reply]
It is available in 2.1. Thank you.

**Comment by t214c** [comment]
I'd like to suggest that the pop up be relative to (just below) the tapped checkmark, rather than in the middle of the screen.

**Reply by hiqua** [reply]
I disagree, having it in the middle of the screen is:
* simpler implementation-wise
* more predictible UX-wise, meaning that e.g. if you tick several habits in a row, you can do it very quickly instead of double-checking where the pop-up appears

**Reply by t214c** [reply]
I would argue that being relative to the primary check mark is also a fixed distance of movement and so probably quicker but I see your reasoning. I was thinking more like the app mentioned in the first post.

It is a small thing anyway.
